src/metadatas.rs

```diff
@@ -144,6 +152,35 @@ pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
     }
 }
 
+impl Metadata {
+    pub fn enrich_with_advanced_infos(&mut self, gtfs: &gtfs_structures::Gtfs) {
+        self.stops_with_wheelchair_info_count = Some(stops_with_wheelchair_info_count(gtfs));
+    }
+}
+
+fn stops_with_wheelchair_info_count(gtfs: &gtfs_structures::Gtfs) -> usize {
+    gtfs.stops
+        .iter()
+        .filter(|(_, stop)| {
+            if stop.wheelchair_boarding != gtfs_structures::Availability::InformationNotAvailable {
+                // information is present
+                return true;
+            } else {
+                // information not present, check for inheritance from parent station
+                let parent_wheelchair_boarding = stop
+                    .parent_station
+                    .as_ref()
+                    .and_then(|parent| gtfs.stops.get(parent))
+                    .and_then(|s| Some(s.wheelchair_boarding));
+
+                // true if parent has information about accessibility
+                return parent_wheelchair_boarding == Some(Availability::Available)
```

Comment:
don't you want to also check `!= gtfs_structures::Availability::InformationNotAvailable` like it's done for the main station (and != None)? but It's mainly the same, so feel free to disregard this comment

Reply:
I found it easier to understand to say information is present if `Availability::Available` or if `Availability::NotAvailable`, than to say not `InformationNotAvailable` and not `None`